src/hooks/useFileManagement.ts

```diff
@@ -3,12 +3,16 @@ import { isValidFileName } from "@/lib/utils/isValidFileName"
 import {
   CodeAndPreviewState,
   CreateFileProps,
+  DeleteFileProps,
 } from "../components/package-port/CodeAndPreview"
+import toast from "react-hot-toast"
+import { useAxios } from "./use-axios"
 
 export function useFileManagement(
   state: CodeAndPreviewState,
   setState: Dispatch<SetStateAction<CodeAndPreviewState>>,
```

Comment:
this is an example of the "leaky abstraction" antipattern, see my other comment for how to fix with an API that's more obvious and doesn't leak state